src/build.d

```diff
@@ -513,6 +514,25 @@ alias checkwhitespace = memoize!(function()
     return new DependencyRef(dep);
 });
 
+alias zip = memoize!(function()
+{
+    Dependency dep;
+    with (dep)
+    {
+        name = "zip";
+        target = "dmdsrc.zip";
+        sources = [thisBuildScript].chain(.sources.root, .sources.backend, .sources.lexer,
```

Comment:
`thisBuildScript` is already an implicit dependency, see
https://github.com/dlang/dmd/blob/442ba964ce07a183ec5d4a6bc13daba083c88bd1/src/build.d#L1346

Reply:
Right, but we still need it in the zip command, I"ll just add it in the command instead then.